hibernate-core/src/main/java/org/hibernate/internal/CoreMessageLogger.java

```diff
@@ -1824,4 +1824,8 @@ void attemptToAssociateProxyWithTwoOpenSessions(
 			id = 514)
 	void postgreSQLJdbcDriverNotAccessible();
 
+	@Message(value = "The identity select string is undefined for the dialect in use. Check the identity generation configuration, in particular the 'hibernate.jdbc.use_get_generated_keys' setting",
```

Comment:
```suggestion
	@Message(value = "Can't retrieve the generated identity value, because the dialect does not support selecting the last generated identity and 'hibernate.jdbc.use_get_generated_keys' was disabled",
```